soc/arm/nxp_lpc/lpc55xxx/soc.h

```diff
@@ -27,10 +27,11 @@
 #define IOCON_PIO_DIGITAL_EN 0x0100u  /*!<@brief Enables digital function */
 #define IOCON_PIO_FUNC0 0x00u         /*!<@brief Selects pin function 0 */
 #define IOCON_PIO_FUNC1 0x01u         /*!<@brief Selects pin function 1 */
-#define IOCON_PIO_FUNC2 0x02u         /*!<@brief Selects pin function 1 */
+#define IOCON_PIO_FUNC2 0x02u         /*!<@brief Selects pin function 2 */
+#define IOCON_PIO_FUNC4 0x04u         /*!<@brief Selects pin function 4 */
 #define IOCON_PIO_FUNC5 0x05u         /*!<@brief Selects pin function 5 */
```

Comment:
is this file something we pull from the SDK? I'm curious as to why we needed to update this file (for future reference)?

Reply:
These defines are from the SDK.